example/webdriverio-test/readme.md

```diff
@@ -43,7 +43,7 @@
    -e JF_WEBRTC_TURN_IP=[ip address] \
    -e JF_WEBRTC_TURN_LISTEN_IP=0.0.0.0 \
    -e JF_SERVER_API_TOKEN=development \
-   ghcr.io/jellyfish-dev/fishjam:0.3.0
+   ghcr.io/fishjam-dev/fishjam:0.3.0
```

Comment:
As you've updated version in `.github/workflows/android_emulator_e2e.yaml`, should it be fixed here too?

Reply:
yep